portality/models/v2/bibjson.py

```diff
@@ -184,7 +184,7 @@ def add_subject(self, scheme, term, code=None):
         self.__seamless__.add_to_list_with_struct("subject", sobj)
 
     @property
-    def apcs(self):
+    def apc(self):
```

Comment:
I'll change this in my branch and update all the downstream stuff like the tests